GenderPayGap.WebUI/Controllers/Account/ChangePasswordNewController.cs

```diff
@@ -86,6 +86,13 @@ public async void ValidatePasswords(ChangePasswordNewViewModel viewModel, User c
                 viewModel.AddErrorFor(m => m.CurrentPassword, "Could not verify your current password.");
             }
 
+            // Check if new password has 8 or more characters
+            if (viewModel.NewPassword.Length < 8)
+            {
+                viewModel.AddErrorFor(m => m.NewPassword, "New password must be at least 8 characters long");
+            }

```

Comment:
This can be done in the ViewModel with `StringLength`, but I felt like it was more readable to have all this password logic in one place so it's obvious what's being done :)